discord/ext/commands/bot.py

```diff
@@ -1033,11 +1033,11 @@ async def process_commands(self, message: Message) -> None:
     async def on_message(self, message):
         await self.process_commands(message)
 
-class Bot(BotBase, discord.Client):
+class Bot(BotBase, discord.Bot):
     """Represents a discord bot.
 
-    This class is a subclass of :class:`discord.Client` and as a result
-    anything that you can do with a :class:`discord.Client` you can do with
+    This class is a subclass of :class:`discord.Bot` and as a result
+    anything that you can do with a :class:`discord.Bot` you can do with
```

Comment:
What's going on here?

Reply:
`discord.Bot` already subclasses `discord.Client` so `commands.Bot` can just subclass `discord.Bot` to get all other features.